src/themes/grayscale.js

```diff
@@ -0,0 +1,145 @@
+export default {
```

Comment:
Please extract shared stuff into variables outside of the export object. I _do_ think having a `shared` namespace in the object would lead to annoyingly complicated merges in the components, but you can DRY up this file some.